activities/ActivityTemplateVue.activity/js/palettes/ballpalette.html

```diff
@@ -0,0 +1,34 @@
+<div class="ball-select">
```

Comment:
Thinks that you could remove this palette and all related images

Reply:
Yes, I'll do that. Will add a palette component later.